contrib/chatops/tests/fixtures.py

```diff
@@ -0,0 +1,69 @@
+__all__ = [
```

Comment:
Not sure if pack tests can use fixtures? I remember seeing a change. I'll have to look into this.